### PR TITLE
💖 feature(documentation): Updated api information

### PR DESCRIPTION
**Pull request by lambro**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -10,26 +10,26 @@ Preparing your code to have it effortlessly work on KintoHub is very straightfor
```
Comment (FanfanDeLioncourt) [inline docs/getting-started.md]: Suggest adding a "Hi there - welcome to KintoHub!" at the top of the page

File: docs/getting-started.md [hunk 2]
```diff
@@ -133,47 +131,46 @@ EXPOSE 80
...
 
-*For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)*
+_For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)_
 
 By this point, you should already have a GitHub repository that is compatible with KintoHub
 
 Next step is connecting your workspace to your GitHub account. Go to edit workspace and click on connect GitHub button.
 
-Now go to create Kintoblock page. Add a Kintoblock name *(name must be unique across all accounts)*, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
+Now go to create Kintoblock page. Add a Kintoblock name _(name must be unique across all accounts)_, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
```
Comment (FanfanDeLioncourt) [inline docs/getting-started.md]: suggest adding all requirements to KB (and KA) creation name here: unique across all accounts, all small caps, no spaces, etc

File: docs/getting-started.md [hunk 3]
```diff
@@ -133,47 +131,46 @@ EXPOSE 80
...
 
-> For the rest of the guide we are going to assume the name is *`nodeexample`*
+> For the rest of the guide we are going to assume the name is _`nodeexample`_
 
-> endpoints are going to be in the following format **http://{microservicename}.api.staging.kintohub.com**
+> endpoints are going to be in the following format **http://api.staging.kintohub.com/{microservicename}**
 
-> Kintoblocks can't be accessed directly, it must be through Applications, if you tried to go to `http://nodeexample.api.staging.kintohub.com/sample/hello` you will get an error.
```
Comment (FanfanDeLioncourt) [inline docs/getting-started.md]: ..."directly. To access them we will need to deploy them through Applications. Therefore, if you tried..."

File: docs/getting-started.md [hunk 4]
```diff
@@ -133,47 +131,46 @@ EXPOSE 80
...
 
-*For more details and screenshots about tagging latest commit read more [here](tagging-a-commit.md)*
+_For more details and screenshots about tagging latest commit read more [here](tagging-a-commit.md)_
 
 After creating the Kintoblock, when you are in the Kintoblock manage page, navigate to the branch that you added your code to. And go to the **Commits** section.
 
-Make sure that the last commit you had is in recent commits and wait until it successfully builds. *(you have to refresh the page to get the latest build status update)*.
-
-After it successfully builds click `Tag Latest Commit` this allows you to add a tag to that commit *(similar to how git works)* so you can select that tag in a Application
+Make sure that the last commit you had is in recent commits and wait until it successfully builds. _(you have to refresh the page to get the latest build status update)_.
```
Comment (FanfanDeLioncourt) [inline docs/getting-started.md]: Can't you do that by using the "Refresh Latest" button we now have?

File: docs/getting-started.md [hunk 5]
```diff
@@ -133,47 +131,46 @@ EXPOSE 80
...
 
+After it successfully builds click `Tag Latest Commit` this allows you to add a tag to that commit _(similar to how git works)_ so you can select that tag in a Application
```
Comment (FanfanDeLioncourt) [inline docs/getting-started.md]: ...you can select that particular tag when you are building an Application (before deploying it)

File: docs/getting-started.md (hunk 1 above)
Comment (lambro) [reply docs/getting-started.md]: cool beans!

File: docs/getting-started.md (hunk 2 above)
Comment (lambro) [reply docs/getting-started.md]: Do you have a list of the requirements? Just to make sure i dont leave something out? 💃

File: docs/getting-started.md (hunk 4 above)
Comment (lambro) [reply docs/getting-started.md]: Yes you can! 👍 I can probably remove this.

Comment (lambro) [reply docs/getting-started.md]: Changed it to 

_(Clicking the "Refresh Latest" button will grab an updated commit list)_

File: docs/getting-started.md (hunk 2 above)
Comment (FanfanDeLioncourt) [reply docs/getting-started.md]: @nadeemkhedr has it - not sure if the excel files have been updated along the last changes

Comment (lambro) [reply docs/getting-started.md]: This is on the dashboard at the moment so I took it from there "name must be unique across all accounts, only lowercase characters and digits are allowed - no spaces and no caps for now please"

**Pull request merged**